procedures/synchronous-meetings.md

```diff
@@ -0,0 +1,87 @@
+
+## Quorum
+
+The meeting has reached quorum if at *most* 2 Council representatives (and their alternates) are not present (i.e., N - 2).
+
+If quorum is not reached, then the sync meeting is canceled and the agenda is gone through asynchronously in chat.
+
+## Minutes
+
+Detailed minutes must be taken of the meeting even if the meeting was otherwise recorded. The meeting minutes must be made public within one week after the meeting concludes (or a delay must be publicly announced and resolved as soon as possible). After the meeting, Council members must review the minutes to ensure that the minutes reflect their understanding of what occurred in the meeting. Once consensus is reached that the minutes are accurate, they are released for public viewing.
+
+Minutes may be found [in the 'minutes' directory of the leadership Council team repository](../minutes).
+
+## Recording
+
+All meetings are recorded, but consensus must be reached after the meeting on what do with the recording. The Council has no obligation to do anything with the recording. If after one week, consensus on what to do with the recording has not been reached, the default is to delete the recording. 
+
+Recordings are usually used to ensure that the notes are correct and to optionally create automatic transcripts. Recordings can also *optionally* be made public, but only with consensus of all of those present on the recording.
+
+## Meeting roles
+
+The meeting has the following assigned roles:
+
+- **Facilitator**: the facilitator is in charge of pushing discussion along, calling on hands, formally asking the group whether there are any objections to a particular motion, and keeping the discussion on agenda.
+- **Scribe(s)**: the scribes are in charge of taking notes on what was discussed in the meeting including noting any action items or follow up questions.
+
+## Meeting Agenda
+
+The facilitator is in charge of ensuring that a proposal for the main agenda for the meeting is made publicly available as a pull request (as the start of the documented minutes for the meeting) to the Council team repository at least 2 days before the meeting. All Council members may comment on, revise, add to or otherwise collaborate on ensuring the agenda is in the right shape.
+
+The agenda should contain the following for each agenda item:
+
+- A short description of the contents of the item
+- The champion of this agenda item
+  - Usually this is the person who has suggested the agenda item and has the most context on what needs to be discussed.
+- An estimated time span for the item
+- A short description of the *goal(s)* of the item
+- An optional proposal (something that requires review and approval through the decision making process)
+  - *Note*: proposals require another Council member to second the proposal before it can be brought forward.
+- Links to previous discussion to facilitate getting up to speed with the current state of discussion around the item.
```

Comment:
As briefly discussed today - some method of pre-raising concerns might be worth adding to agenda to identify stakeholders and topics that need discussion.